lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/Screen.js

```diff
@@ -96,6 +96,44 @@ qx.Class.define("callbackery.ui.Screen", {
                     that.setLayout(lc);
                     content.set(containerSet);
                     that.add(content,containerAddProps);
+
+                    // Show (legal) disclaimer and hide screen content until
+                    // checkbox is set and btn is executed.
+                    var disclaimerCfg = options.disclaimer;
+                    if (disclaimerCfg) {
+                        content.hide();
+                        // force widget visibility because it is tied to
+                        // content visibility above
+                        that.setVisibility('visible');
+
+                        var disclaimerContainer = new qx.ui.container.Composite(new qx.ui.layout.VBox(10));
+                        var disclaimer = new qx.ui.basic.Label(that.xtr(disclaimerCfg.note)).set({
+                            rich: true
+                        });
+
+                        var btnRow     = new qx.ui.container.Composite(new qx.ui.layout.HBox(10, 'right'));
+
+                        var okBtnLabel = disclaimerCfg.okButtonLabel ? that.xtr(disclaimerCfg.okButtonLabel)
+                                                                     : that.xtr('OK');
+                        var okBtn = new qx.ui.form.Button(okBtnLabel).set({enabled : false});
+                        okBtn.addListener('execute', function() {
+                            content.show();
+                            disclaimerContainer.hide();
+                        }, that);
+                        btnRow.add(okBtn);
+
+                        var check = new qx.ui.form.CheckBox(that.xtr(disclaimerCfg.label)).set({
+                            rich: true
+                        });
+                        check.addListener('changeValue', function(e) {
+                            okBtn.setEnabled(e.getData());
+                        }, that);
```

Comment:
you can use single value binding to bind the checkbox and the okButton visiblity

Reply:
Bound to `enabled`.